### PR TITLE
Add flox

### PR DESCRIPTION
**Pull request by dcherian**
**I have renamed the `dask_groupby` package to `flox`** please delete the old feedstock (cc @dopplershift)

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/flox/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "flox" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dask_groupby-{{ version }}.tar.gz
```
Comment (dopplershift) [inline recipes/flox/meta.yaml:10]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
?

File: recipes/flox/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - dask-core
+    - importlib-metadata
+    - numpy_groupies
+    - python >=3.7
+    - toolz
+    - xarray
+
+test:
+  imports:
+    - dask_groupby
```
Comment (dopplershift) [inline recipes/flox/meta.yaml:33]:
```suggestion
    - flox
```
?

**Comment by jakirkham** [comment]
Fitting name give this discussion ( https://github.com/dask/community/issues/204#issuecomment-966654238 ) 🙂

**Comment by dcherian** [comment]
ouch thank you. I thought I had done a find-replace but I guess not...

> Fitting name give this discussion ( dask/community#204 (comment) )

Yes! Hopefully with this package you wont; need a flock of phoenixes, sheep will work just fine ;)

File: recipes/flox/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a3766df3775caf44be2561b50736e9634f6326b2e4c9c615f5a0728de3a554e1
```
Comment (dcherian) [inline recipes/flox/meta.yaml:11]:
```suggestion
  sha256: bbcab816249795d7a23e42b37f39c2ff04bf789c768a1a0a2551fe9d3d6c20fd
```

**Review by dopplershift — APPROVED** [review]

**Pull request merged**